api/src/main/java/org/openmrs/module/metadatamapping/api/MetadataTermMappingSearchCriteria.java

```diff
@@ -11,8 +11,14 @@
  */
 public class MetadataTermMappingSearchCriteria {
 	
+	public enum DefinedOrUndefined {
+		TRUE, FALSE, NULL
+	}
+	
 	private boolean includeAll;
 	
+	private DefinedOrUndefined definedOrUndefined = DefinedOrUndefined.NULL;
```

Comment:
@rkorytkowski 
I think there is need to use enum since we need to choose between 3 states of MetadataTermMapping's to recieve:

- mapped only <`mapped=false`>,
- not mapped only <`mapped=true`> 
- all mappings<`mapped=null`>

If I used boolean type here, MetadataTermMappingSearchCriteriaBuilder would use default boolean value if there is `mapped` param is undefined, so query would return only not mapped mappings only.